docs/cookbook/configuration.rst

```diff
@@ -89,6 +89,19 @@ Some examples:
         #     namespace: The\Namespace
         my_suite: The\Namespace
 
+.. tip:: You may use ``%paths.config%`` in ``src_path`` and ``spec_path`` making paths relative to the location of the config file.
```

Comment:
You could mention that by default they're relative to the current working directory